docs/en/user_guides/classification.md

```diff
@@ -1,20 +1,19 @@
 # Classification
 
-In MMSelfSup, we provide many benchmarks, thus the models can be evaluated on different downstream tasks. Here are comprehensive tutorials and examples to explain how to run all benchmarks with MMSelfSup.
-
 - [Classification](#classification)
   - [VOC SVM / Low-shot SVM](#voc-svm--low-shot-svm)
   - [Linear Evaluation](#linear-evaluation)
```

Comment:
TOC has not been updated, now the subtitle is `Linear Evaluation and fine-tuning`

Reply:
> TOC has not been updated, now the subtitle is `Linear Evaluation and fine-tuning`
done